tests/components/wiz_light/test_config_flow.py

```diff
@@ -0,0 +1,158 @@
+"""Test the WiZ Light config flow."""
+from homeassistant import config_entries, setup
+from homeassistant.components.wiz_light.config_flow import (
+    WizLightConnectionError,
+    WizLightTimeOutError,
+)
+from homeassistant.components.wiz_light.const import DOMAIN
+from homeassistant.const import CONF_HOST, CONF_NAME
+
+from tests.async_mock import patch
```

Comment:
https://github.com/home-assistant/core/pull/44746

Change to: from unittest.mock import patch

Reply:
Thanks for the input. I've changed it to the native input. I assume hat the `tests.async_mock` dispatcher will be obsolete int the future. I've committed the change ... lets see if it will pass 😄